.github/workflows/unit-test-coverage.yml

```diff
@@ -1,9 +1,12 @@
-name: Go
-
+name: Unit Test Coverage
 
 on:
   pull_request:
-    branches: [ master ]
+  push:
+    branches:
+      - master
+      - release/**
```

Comment:
I think the issue is that we're not running on the master branch (base commit that PRs are forked from)

![image](https://github.com/sei-protocol/sei-chain/assets/18161326/ab2c8c0f-e5ef-4d53-83ac-2e70334a0707)